website/docs/package.json

```diff
@@ -1,61 +1,39 @@
 {
-  "name": "luigi-documentation",
-  "description": "Interactive documentation based on svelte3, sapper and luigi",
-  "private": true,
-  "scripts": {
-    "start": "run-p dev:*",
-    "dev:sapper": "PORT=4001 sapper dev",
-    "dev:config": "npm run build:config -- --watch",
-    "dev:luigi": "sirv static/luigi --single --dev --cors --port 4000 --host 0.0.0.0",
-    "debug:sapper": "PORT=4001 node --inspect node_modules/sapper/dist/cli.js dev",
-    "fetchversions": "curl -H \"Accept: application/vnd.github.v3+json\" https://api.github.com/repos/SAP/luigi/tags?per_page=100 > static/versions.json",
-    "build:config": "npm run fetchversions && node -r esm scripts/generate-docu-sitemap.js && webpack --config webpack-generateConfig.config",
-    "export:sapper": "NODE_ENV=production sapper export --basepath /docu-microfrontend && sh ./scripts/move-export.sh",
-    "export:serve": "sirv public --single --dev --cors --port 5000 --host 0.0.0.0",
-    "export": "npm run build:config && npm run export:sapper",
-    "mock:develop-plugins": "NODE_ENV=debug node -r esm scripts/markdown.unified-preview.js",
-    "mock:develop-plugins:watch": "NODE_ENV=debug nodemon --watch src --watch scripts -e js,md -r esm scripts/markdown.unified-preview.js"
```

Comment:
Seems like there is plenty of scripts that were gone here. 
For example **./scripts/generate-docu-sitemap.js** has to be run somewhere. If this is deleted here and not added somewhere else then we won't be having any docu sitemap. Also the **fetchversions** seems to fetch latest luigi from the repo to get the latest docu updates. That also needs to stay there. Somehow these configs shouldn't be deleted but translated to vite/sveltekit corresponding calls